support-frontend/assets/helpers/tracking/quantumMetric.ts

```diff
@@ -143,57 +143,26 @@ function sendEventAcquisitionDataFromQueryParamEvent(
 ): void {
 	void ifQmPermitted(() => {
 		const sendEventWhenReady = () => {
-			const source = acquisitionData.source;
-			const componentId = acquisitionData.componentId;
-			const componentType = acquisitionData.componentType;
-			const campaignCode = acquisitionData.campaignCode;
-			const referrerUrl = acquisitionData.referrerUrl;
-			const isRemote = acquisitionData.isRemote;
-
-			if (window.QuantumMetricAPI?.isOn()) {
-				if (source) {
-					sendEvent(
-						SendEventAcquisitionDataFromQueryParam.Source,
-						false,
-						source.toString(),
-					);
-				}
-				if (componentId) {
-					sendEvent(
-						SendEventAcquisitionDataFromQueryParam.ComponentId,
-						false,
-						componentId.toString(),
-					);
-				}
-				if (componentType) {
-					sendEvent(
-						SendEventAcquisitionDataFromQueryParam.ComponentType,
-						false,
-						componentType.toString(),
-					);
-				}
-				if (campaignCode) {
-					sendEvent(
-						SendEventAcquisitionDataFromQueryParam.CampaignCode,
-						false,
-						campaignCode.toString(),
-					);
-				}
-				if (referrerUrl) {
-					sendEvent(
-						SendEventAcquisitionDataFromQueryParam.ReferrerUrl,
-						false,
-						referrerUrl.toString(),
-					);
-				}
-				if (isRemote) {
+			type ReferrerAcquisitionDataKeysToLogType = Record<string, number>;
```

Comment:
Very minor, possibly just call type `ReferAcquisitionDataKeys` might be shorter and might be a logical extension of type `ReferAcquisitionData` used within this call.

Reply:
Like the way you did this